types/src/generator/index.ts

```diff
@@ -0,0 +1,183 @@
+import assert from "assert";
```

Comment:
@kentonv maybe we want this in `src/types`? or `src/workerd/types`?

Reply:
@kentonv .. just pinging again on this.

However, I don't think answering this should block this PR landing because we can always move this if necessary.